extension/opampextension/config.go

```diff
@@ -123,6 +132,14 @@ func (s OpAMPServer) GetEndpoint() string {
 	return ""
 }
 
+func (s OpAMPServer) GetPollingInterval() time.Duration {
+	if s.HTTP != nil {
+		return s.HTTP.PollingInterval
+	}
```

Comment:
If a polling interval is not set in the config, PollingInterval would be 0 here. I think if polling interval is unset/0, we should use some default interval (e.g. 30 seconds, like opamp-go does by default)

Reply:
We should be getting the default from the opamp-go in that case, but if you prefer to do it explicitly on our side I can do that.